### PR TITLE
Log the number of Streams since it might be different than the number of Threads

### PR DESCRIPTION
**Pull request by gartung**
I found this to be missing when running benchmarks on the KNL system.

### CODE REVIEW

**Comment by gartung** [comment]
@Dr15Jones @smuzaffar

File: FWCore/Framework/src/EventProcessor.cc [hunk 1]
```diff
@@ -434,6 +434,8 @@ namespace edm {
         nStreams = nThreads;
       }
     }
+    // PG: Log the number of streams
+    edm::LogInfo("StreamSetup") <<"setting # streams "<<nStreams;
```
Comment (Dr15Jones) [inline FWCore/Framework/src/EventProcessor.cc]: Only print number of streams if the option was explicitly set. That would be moving this to be between line 435 and 436.

**Comment by gartung** [comment]
@Dr15Jones  OK I moved the logging location.

**Comment by Dr15Jones** [comment]
please test

**Comment by Dr15Jones** [comment]
+1

**Comment by gartung** [comment]
@Dr15Jones please retest.

**Comment by Dr15Jones** [comment]
please test

**Comment by Dr15Jones** [comment]
+1

**Comment by smuzaffar** [comment]
@davidlange6 , bot adds ORP signatures for release based on normal branch (e.g. CMSSW_8_1_X). This PR is for DEVEL IB that is why bot did not add ORP. Should I change it so that it adds ORP signs for all PRs?

**Comment by davidlange6** [comment]
ah - I missed that.. yes- lets add orp for the devel build as well as we will build the next release cycle from it

**Pull request merged**